cmd/promtool/main.go

```diff
@@ -501,7 +501,8 @@ func QueryLabels(url *url.URL, name string, p printer) int {
 	// Run query against client.
 	api := v1.NewAPI(c)
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	val, err := api.LabelValues(ctx, name)
+	// TODO: Show warnings (second return value) somehow?
```

Comment:
Can we add an issue for this? Otherwise, it will get lost.

Reply:
Created https://github.com/prometheus/prometheus/issues/5885 and added to comment here.

Now still have to figure out why a logger is `nil` in web tests...